src/servers/ZoneServer2016/zoneserver.ts

```diff
@@ -420,7 +436,7 @@ export class ZoneServer2016 extends ZoneServer {
             };
           }),
         },
-        recipes: recipes,
+        recipes: Object.values(this._recipes),
```

Comment:
this value is static like recipes never changes so `Object.values(this._recipes)` should be stored in a variable

Reply:
recipes will eventually be stored in each character class for whatever recipes they have discovered, so this is just temporary.